Paco/assets/custom_base.js

```diff
@@ -243,6 +253,10 @@ var paco = (function (init) {
         $.each(db.getAllEvents(), function(index, value) { newarray[index] = value });
         return newarray;
     };
+    
+    var getEventsByQuery = function(jsonText){
```

Comment:
change the parameter name to something more intention revealing such as queryJson. Also, it would be good to add a comment above the method that describes the format of the parameter.